wagtail/tests/test_management_commands.py

```diff
@@ -697,6 +698,44 @@ def test_purge_revisions_with_date_cutoff(self):
         )
 
 
+class TestPurgeEmbedsCommand(TestCase):
+    fixtures = ["test.json"]
+
+    def setUp(self):
+        # create dummy Embed objects
+        for i in range(5):
+            embed = Embed(
+                hash=f"{i}",
+                url="https://www.youtube.com/watch?v=Js8dIRxwSRY",
+                max_width=None,
+                type="video",
+                html="test html",
+                title="test title",
+                author_name="test author name",
+                provider_name="test provider name",
+                thumbnail_url="http://test/thumbnail.url",
+                width=1000,
+                height=1000,
+            )
+            embed.save()
+
+    def test_purge_embeds(self):
+        """
+        fetch all dummy embeds and confirm they are deleted when the management command runs
+
+        """
+
+        embeds = Embed.objects.all()
+
+        self.assertEqual(embeds.count(), 5)
+
+        management.call_command("purge_embeds", stdout=StringIO())
+
+        embeds = Embed.objects.all()
```

Comment:
Nit: This line isn't necessary, and the test will still work.

Suggestion: Inline creating the queryset into the assertion, eg `self.assertEqual(Embed.objects.count(), 5)`.